### PR TITLE
fix: add abhinavkashyap061

### LINKED ISSUE #1142: Abhinav Kashyap
## Profile Screenshot

image
- PS: it is an animated banner though

### PR DESCRIPTION
**Pull request by abhinavkashyap061**
resolves #1142 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/abhinavkashyap061.json
Comment (Panquesito7) [inline profiles/abhinavkashyap061.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by abhinavkashyap061** [comment]
- ok thanks! do I need to do anything else?? i am a newbie open source contributor 😅

**Comment by schmelto** [comment]
> * ok thanks! do I need to do anything else?? i am a newbie open source contributor 😅

No all good. Just wait for another maintainer to approve the PR and than this will get merged.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**
